### PR TITLE
Added EBS volumes to Instances

### PR DESCRIPTION
**Pull request by Jan-M**
Needed for EBS credit monitoring

### CODE REVIEW

File: zmon_aws_agent/aws.py [hunk 1]
```diff
@@ -192,6 +192,15 @@ def get_running_apps(region):
                 'infrastructure_account': 'aws:{}'.format(owner),
             }
 
+            i["block_devices"] = {}
+            for device in i.get("BlockDeviceMappings", []):
+                if "Ebs" in device:
+                    i["block_devices"][device["DeviceName"]] = {
+                        "volume_id": device["Ebs"]["VolumeId"],
+                        "volume_type":"ebs",
```
Comment (hjacobs) [inline zmon_aws_agent/aws.py]: missing space after ":"

**Comment by Jan-M** [comment]
:+1:

**Comment by hjacobs** [comment]
:+1:

File: zmon_aws_agent/aws.py [hunk 2]
```diff
@@ -192,6 +192,15 @@ def get_running_apps(region):
                 'infrastructure_account': 'aws:{}'.format(owner),
             }
 
+            i["block_devices"] = {}
+            for device in i.get("BlockDeviceMappings", []):
+                if "Ebs" in device:
+                    i["block_devices"][device["DeviceName"]] = {
+                        "volume_id": device["Ebs"]["VolumeId"],
+                        "volume_type": "ebs",
```
Comment (alexeyklyukin) [inline zmon_aws_agent/aws.py]: is there a device['Ebs']['VolumeType']? We need it in order to filter only gp2 ones.

Comment (alexeyklyukin) [inline zmon_aws_agent/aws.py]: VolumeSize would be nice to have as well

**Pull request merged**